_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
Cool, ya I'll change it to `*-styleguide`. Will let you know when it's ready to go.